jira_agile_metrics/calculators/cycletime_test.py

```diff
@@ -88,6 +88,27 @@ def jira(custom_fields):
         ),
     ])
 
+@pytest.fixture
+def jira_with_skipped_columns(custom_fields):
+    return JIRA(fields=custom_fields, issues=[
+        Issue("A-10",
+            summary="Gaps",
+            issuetype=Value("Story", "story"),
+            status=Value("Done", "done"),
+            resolution=Value("Done", "Done"),
+            resolutiondate="2018-01-04 01:01:01",
+            created="2018-01-01 01:01:01",
+            customfield_001="Team 1",
+            customfield_002=Value(None, 10),
+            customfield_003=Value(None, []),
+            customfield_100=None,
+            changes=[
+                Change("2018-01-02 01:05:01", [("status", "Backlog", "Next",)]),
```

Comment:
This is the test I think that needs to be changed to reproduce the issue I'm talking about. If you change the transition from Backlog to Build, I think the test will fail, while the age of the ticket hasn't really changed (it's started work on 2018-01-02, though the first stage was Build and not Committed).